src/services/subscription.service.js

```diff
@@ -45,6 +77,8 @@ module.exports.create = (_args, { req }) => {
       return;
     }
     if (!resource.id) resource.id = uuidv4();
+    resource.status = 'active';
+    resource.numEventsSinceStart = 0;
```

Comment:
For just storing this in the db this is fine, but when we return the subscription via search this `numEventsSinceStart` is a non-standard field. Do you know if FHIR tools tend to break if there's a nonstandard field included? Tbh I don't expect any issues in actual use so I'm not too worried either way

Reply:
I forget which environment I was in (probably HAPI) that I had sent a FHIR resource with nonstandard field included and it had a warning about it, but didn't break. I would be surprised if anything broke. I can remove it in the search methods though if we think it will be a problem